gsa/src/web/pages/targets/component.js

```diff
@@ -56,43 +56,47 @@ const TargetComponent = props => {
   const [portListDialogVisible, setPortListDialogVisible] = useState(false);
   const [targetDialogVisible, setTargetDialogVisible] = useState(false);
 
+  const [targetDialogState, setTargetDialogState] = useState({
+    id: undefined,
```

Comment:
You can skip the undefined object properties if you want. It is not necessary to initialize these properties. They are undefined by default.